docs/sources/auth/generic-oauth.md

```diff
@@ -38,10 +38,17 @@ token_url =
 api_url =
 allowed_domains = mycompany.com mycompany.org
 allow_sign_up = true
+tls_skip_verify_insecure = false
+tls_client_cert =
+tls_client_key =
+tls_client_ca =
 ```
 
 Set `api_url` to the resource that returns [OpenID UserInfo](https://connect2id.com/products/server/docs/api/userinfo) compatible information.
 
+You can also specify the TLS configuration used by the client. Set `tls_client_cert` to the path of the certificate, `tls_client_key` to the path containing the key and `tls_client_ca` to the path containing a trusted certificate authority list. 
```

Comment:
Didn't we agree at some point to use the term SSL/TLS, since they're synonymous and SSL is the more popular term these days?

Reply:
ok, I hardly remember having this discussion.
do you suggest to change the documentation or the settings as well?
If the latter, changing the settings would be breaking change; it would require users to update their configurations. Anyway I would suggest updating the documentation fo now to be compliant with the existing implementation and decide about the configuration names later.

Reply:
@papagian See discussion [here](https://github.com/grafana/grafana/pull/21341#discussion_r364120835). I think we've decided to standardize on "SSL", but Diana wanted us to include TLS as an alternate term so people can recognize it.